magenta/interfaces/midi/README.md

```diff
@@ -130,24 +130,40 @@ magenta_midi --list_ports
 
 You should see a list of available input and output ports, including both the
 controller (e.g., "VMPK Output") and synthesizer (e.g., "FluidSynth virtual
-port").
+port"). Set the environment variables based on the ports you want to use. For
+example:
 
-To use the midi interface, you must supply a trained model bundle (.mag file).
-You can either download one from the links on our model pages (e.g.,
-[Melody RNN](/magenta/models/melody_rnn/README.md) or create a bundle file from
-one of your training checkpoints using the instructions on the model page.
+```bash
+CONTROLLER_PORT="VMPK Output"
+SYNTH_PORT="FluidSynth virtual port 1"
+```
+
+To use the midi interface, you must supply one or more trained model bundles
+(.mag files). You can either download them from the links on our model pages
+(e.g., [Melody RNN](/magenta/models/melody_rnn/README.md)) or create bundle
+files from your training checkpoints using the instructions on the model page.
+Once you're picked out the bundle files you wish to use, set the magenta_midi --help
+environment
+variable with a comma-separated list of paths to to the bundles. For example:
+
+```bash
+BUNDLE_PATHS=/path/to/bundle1.mag,/path/to/bundle2.mag
+```
 
-You will now start the interface with this command, supplying the location of
-the .mag bundle file and any additional flags required by the interaction (see
-below):
+You may now start the interface with this command:
 
 ```bash
 magenta_midi \
```

Comment:
I think it may be helpful to include setting the environment variables here again, just so people don't have to read through all the text above to figure out what they should look like